zerver/tests/test_events.py

```diff
@@ -2781,7 +2783,7 @@ def test_do_delete_message_no_max_id(self) -> None:
             lambda: do_delete_messages(self.user_profile.realm, [message]),
             state_change_expected=True,
         )
-        result = fetch_initial_state_data(user_profile, None, "", client_gravatar=False)
+        result = fetch_initial_state_data(user_profile, None, "", client_gravatar=False, exclude_old_users_avatars=False)
```

Comment:
Is there any need to add something here?  I'd have expected the default to apply without any need to patch this.

Reply:
@timabbott I haven't added a default value for this parameter, that's why we need it here. Should I add it?